internal/cmd/commands/clientagentcmd/wrapper_register.go

```diff
@@ -5,14 +5,23 @@ package clientagentcmd
 
 import (
 	"context"
+	"os"
+	"strings"
 
 	"github.com/hashicorp/boundary/internal/cmd/base"
 	"github.com/hashicorp/boundary/internal/cmd/wrapper"
 )
 
-// TODO (ICU-13140): Remove this and re-enable error output for background
-// client agent token sending.
-const allowErrorOutput = false
+var allowErrorOutput = false
+
+const EnvBoundaryClientAgentCliErrorOutput = "BOUNDARY_CLIENT_AGENT_CLI_ERROR_OUTPUT"
+
+func init() {
+	errOutput := os.Getenv(EnvBoundaryClientAgentCliErrorOutput)
+	if strings.ToLower(errOutput) == "true" {
+		allowErrorOutput = true
+	}
+}
```

Comment:
It would be nice if we could pass this down through flags like other configuration values. What do you think?

Reply:
Sure, I'll do that. I'll open a PR for those changes